AGameBase/AGameBase.cpp

```diff
@@ -45,21 +45,20 @@ void AGameBase::Tick(float deltaTime) {
     g_pGlobals->update();
 }
 
-UWorld* g_pWorld = NULL;
-void    AGameBase::BeginPlay() {
+void AGameBase::BeginPlay() {
     Super::BeginPlay();
-    ADebug::Assert((bool)(s_iEntityCount = g_entList.Num()),
-                   "\nIBaseEntity::s_iEntityCount = g_entList.Num()");
+
+    ADebug::Assert(s_iEntityCount == g_entList.Num(),
+                   "\nIBaseEntity::s_iEntityCount == g_entList.Num()");
```

Comment:
This assert statement was written incorrectly and I didn't catch it the first time